osi_object.proto

```diff
@@ -512,6 +512,65 @@ message MovingObject
         TYPE_ANIMAL = 4;
     }
 
+    //
+    // \brief Additional pedestrian data in case the moving object is a human.
+    //
+    // This is an extension to the \c MovingObject with additional information 
+    // describing a pedestrian in more detail.
+    // 
+    message PedestrianData {
+
+        // Body height of the pedestrian  
+        // 
+        // Unit: m
+        //
+        optional double height = 1;
+
+        // List of all skeleton points of the pedestrian
+        // 
+        // The number of skeleton points may very, based on the detail level of 
+        // the pedestrian model used. For example, some simulators will not include 
+        // detailed data of the fingers or eyes of the pedestrian.
+        // 
+        repeated SkeletonPoint skeleton_data = 2;
+    }
+
+
+    // 
+    // \brief Points in the skeleton of the pedestrian
+    //
+    // Each point represents a joint, or otherwise important point in the skeleton 
+    // of a pedestrian. For example Pelvis, Knee or Shoulder. The naming convention 
+    // should be followed for identifying skeleton points.
+    //
+    message SkeletonPoint {
+
+
+        // Skeleton points are identified by a naming scheme
+        //  
+        // To properly identify the skeleton points the pre-defined naming convention 
+        // must be used.
+        // 
+        //\image html OSI_SkeletonNamingConvention.jpg
+        // 
+        required string identifier = 1;
+
+        // Position of the skeleton point
+        // 
+        // Reference System is the middle of the bounding box (\c MovingObject::base 
+        // \c BaseMoving::position).
+        // 
+        required Vector3d position = 2;
+
+        // Orientation of the skeleton point
+        //
+        // Reference System is the middle of the bounding box (\c MovingObject::base 
+        // \c BaseMoving::position).
```

Comment:
This confuses me. It mentions BaseMoving::position, which shouldn't have any impact on the orientation. I assume that it is relative to BaseMoving::orientation? Or is the orientation absolute?

Reply:
Yes, this is a copy-paste error in the comment. The correct reference would be BaseMoving::orientation. Will be fixed.